manage_cortex_prs.sh

```
@@ -0,0 +1,435 @@
+    echo ""
+    sleep 1
+done
+
+echo ""
+echo "⏰ STEP 6: SENDING STALE PR REMINDERS"
+echo "====================================="
+echo ""
+
+for pr_num in "${stale_prs[@]}"; do
+    # Skip if it's in draft or critical (already handled)
+    if [[ " ${draft_prs[@]} " =~ " ${pr_num} " ]] || [[ " ${critical_prs[@]} " =~ " ${pr_num} " ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)